ui/page/main_page.go

```diff
@@ -748,7 +748,7 @@ func (mp *MainPage) LayoutUSDBalance(gtx layout.Context) layout.Dimensions {
 	}
 }
 
-func (mp *MainPage) totalDCRBalance(gtx C) D {
+func (mp *MainPage) TotalDCRBalance(gtx C) D {
```

Comment:
Why do we have this as a global variable when it is only used on this page? Why was this change made? Kindly revert if it does not impact your PR.

Reply:
@devchoplife  please revert it